### PR TITLE
Swg entitlements banner

### PR DESCRIPTION
**Pull request by jkerr321**
# Feature Description
Google have asked that we add entitlements checking to Article pages.

# Link to Ticket / Card:
https://trello.com/c/WgGJ6ueb

## Image
image

## Related PRs
https://github.com/Financial-Times/n-swg/pull/69
https://github.com/Financial-Times/next-ammit-api/pull/472

### CODE REVIEW

File: demos/app.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ app.get('/*', (req, res) => {
 	res.render('index',{ layout: 'custom-vanilla', title: 'Demo' });
 });
 
-const PORT = process.env.PORT || 5005;
+const PORT = process.env.PORT || 5050;
```
Comment (jkerr321) [inline demos/app.js]: required as 5050 is on google's whitelist and 5005 is not. Do we care about this? Should I change it back? (we have asked google to whitelist 5005 as well but not heard back yet as far as I know - @remybach)?

Comment (remybach) [reply demos/app.js]: Google have confirmed that it's been done if you want to test it.

File: src/components/bottom/swg-entitlements-prompt/main.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+const swgImport = require('n-swg');
+const swgLoader = swgImport.swgLoader;
+
+module.exports = (banner) => {
+		let swg;
+		swgLoader({ manualInitDomain: 'ft.com:subscribed', customOnwardJourney: true })
```
Comment (remybach) [inline src/components/bottom/swg-entitlements-prompt/main.js]: Ah, just realised... it's only for the demo page that you'll need the `manualInitDomain` bit.

File: demos/app.js (hunk 1 above)
Comment (jkerr321) [reply demos/app.js]: Brill, thanks!

File: src/components/bottom/swg-entitlements-prompt/main.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+const swgImport = require('n-swg');
```
Comment (liamkeaton) [inline src/components/bottom/swg-entitlements-prompt/main.js]: This could be `require('n-swg').swgLoader`

**Pull request merged**